lib/src/lib/page-header/components/page-header/page-header.component.html

```diff
@@ -30,8 +30,14 @@ <h1 *ngIf="pageTitle">{{ pageTitle | translate: pageTitleArgs }}</h1>
         [queryParams]="link?.queryParams"
         #rla="routerLinkActive"
         [active]="rla.isActive"
-        >{{ getLabel(link) | translate }}</a
+        class="lab900-page-header__nav__item"
       >
+        <mat-icon class="lab900-page-header__nav__item__prefix-icon" *ngIf="link.svgIcon" [svgIcon]="link.prefixIcon"></mat-icon>
+        <mat-icon class="lab900-page-header__nav__item__prefix-icon" *ngIf="!link.svgIcon">{{ link.prefixIcon }}</mat-icon>
```

Comment:
if there is no prefixIcon or suffixIcon set, you will still load an empty mat-icon component. Wouldn't it be better to wrap this in another ngIf on prefixIcon and the same for suffixIcon?